en/docs/install-and-setup/perfromance-tuning-and-test-results/tuning-performance.md

```diff
@@ -445,7 +448,7 @@ The Thrift section is related to Analytics and the Binary section is related to
 <td><code>             batch_size            </code></td>
 <td>The WSO2 API-M statistical data sent to the Analytics Server to be published in the Analytics Dashboard are grouped into batches. This parameter specifies the number of requests to be included in a batch.</td>
 <td>200</td>
-<td>This value should be tuned in proportion to the volume of requests sent from WSO2 API-M to the Analytics Server. This value should be reduced if you want to reduce the system overhead of the Analytics Server. This value should be increased if WSO2 API-M is generating a high amount of statistics and if the <code>             queue_size            </code> cannot be further increased without causing an overconsumption of memory.</td>
+<td>This value should be tuned in proportion to the volume of requests sent from WSO2 API-M to the Analytics Server. This value should be reduced if you want to reduce the system overhead of the Analytics Server. This value should be increased if WSO2 API-M is generating a high amount of statistics and if the <code>             queue_size            </code> cannot be further increased without causing overconsumption of memory.</td>
```

Comment:
"This value" is repeated in each sentence. Can we use some other way to convey the message?

Reply:
Done